purchases/src/testDefaults/kotlin/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -114,7 +114,7 @@ internal class PurchasesTest : BasePurchasesTest() {
     // region purchasing
 
     @Test
-    fun `deprecated upgrade defaults ProrationMode to null if not passed`() {
+    fun `upgrade defaults ReplacementMode to WITHOUT_PRORATION if not passed`() {
```

Comment:
I changed this test to assert the current behavior, assuming that that behavior is correct.